WinCompatibilityPack/WinCompatibilityPack.csproj

```diff
@@ -9,7 +9,5 @@
     <PackageReference Include="PowerShellStandard.Library" Version="5.1.0-*">
```

Comment:
Why do we need to reference `PowerShellStandard.Library`? This project is only for gathering WCP assemblies, we don't really build anything that depends on `PowerShellStandard.Library`.